### DISCUSSION TITLE
Ansible role for Opencanary

### DISCUSSION THREAD

**Discussion by theidiotyouyellat** [thinkst/opencanary#288] [Show and tell]
First off, I would like to thank the folks at Thinkst Canary for continued support for this project. I have been using this in production alongside the Thinkst Canary product with good success. I wanted to give back to the project in some way and since my python skills are rudimentary at best, I decided to offer up my ansible role I created to help build / reconfigure opencanaries.

I have done testing mostly on Ubuntu/Debian with minimal tests on CentOS Stream/RHEL.
[ansible-role-opencanary](https://github.com/theidiotyouyellat/ansible-role-opencanary)

Hope this helps those wanting to deploy this project. Cheers!

**Comment by jayjb** [comment]
Hi @theidiotyouyellat,

Sorry for the delay! Thanks for the kind words and we are so glad that it is working for you. Any and all help is always appreciated. 

My Ansible skills are, as you put it rudimentary at best, so thank you very much for supplying this. Would you like to maybe throw it into a folder in the OpenCanary repo. Something like `automations` and then we have `ansible`.

**Reply by theidiotyouyellat** [reply]
I'm not sure it makes sense to put this in the opencanary repo, but not opposed to the idea. 

My thought is that you clone/download the ansible role to your ansible controller then use the controller to install/configure other systems to run OpenCanary.

**Comment by jayjb** [comment]
You could submit it as a PR and we take your documentation and add it to the docs too. Let me know if you would be keen to do that and we can get it sorted together.

**Comment by jayjb** [comment]
Hi @theidiotyouyellat,

I've chatted to some of the folks here and we thought that maybe we could fork your repo and reference it as the source. The reason we would like to fork it is that if you ever decide to archive or kill it, we would still have our own copy. Does that sound reasonable to you?

**Reply by theidiotyouyellat** [reply]
I have no issue with that. Please take and use how you see fit.

**Comment by jayjb** [comment]
Thanks @theidiotyouyellat! We've forked it and referenced it in the OpenCanary documentation. Please email me at: jay [at] thinkst [dot] com because we would like to send you something.
